docs/en/operations/system-tables.md

```diff
@@ -1335,4 +1335,95 @@ Columns:
 
 If the storage policy contains more then one volume, then information for each volume is stored in the individual row of the table.
 
+## system.quotas {#system_tables-quotas}
+Contains information about [quotas](quotas.md).
+
+Columns:
+-  `name` ([String](../sql-reference/data-types/string.md)) — Quota name.
+-   `id` ([UUID](../sql-reference/data-types/uuid.md)) — Quota ID.
+-   `storage`([String](../sql-reference/data-types/string.md)) — Storage of quotas. Possible value: "users.xml" if a quota configured in the users.xml file, "disk" if a quota configured by an SQL-query.
+-   `key_type` ([Enum8](../sql-reference/data-types/enum.md)) — The type of key that the quota is tracked for. Values:
+    -   `'none' = 0` — All users share the same quota.
+    -   `'user name' = 1` — Connections with the same user name share the same quota.
+    -   `'ip address' = 2` — Connections from the same IP share the same quota.
+    -   `'client key' = 3` — Connections with the same key, which explicitly provide by client, share the same quota. When using clickhouse-client, this is the `--quota-key` parameter value from the command line, or the `quota_key` value from the client config. If using HTTP interface, you can specify the "X-ClickHouse-Quota" parameter in the request header.
+    -   `'client key or user name' = 4` — Uses `client key`, but if the client doesn't provide a key, `user name` uses.
+    -   `'client key or ip address' = 5` — Uses `client key`, but if the client doesn't provide a key, `ip_address` uses.
+-   `roles` ([Array](../sql-reference/data-types/array.md)([String](../sql-reference/data-types/string.md))) — Name of the [roles](../operations/access-rights.md#role-management) or users should use this quota.
+-   `intervals.duration` ([Array](../sql-reference/data-types/array.md)([UInt64](../sql-reference/data-types/int-uint.md))) — Length of the time intervals, in seconds.
+-   `intervals.randomize_interval` ([Array](../sql-reference/data-types/array.md)([UInt8](../sql-reference/data-types/int-uint.md#uint-ranges))) — Logical value, it is shows whether the interval is randomized. Interval always starts at the same time if it is not randomized. For example, an interval of 1 minute always starts at an integer number of minutes (i.e. it can start at 11:20:00, but it never starts at 11:20:01), an interval of one day always starts at midnight UTC. The randomized interval starts anytime. Values:
```

Comment:
```suggestion
-   `intervals.randomize_interval` ([Array](../sql-reference/data-types/array.md)([UInt8](../sql-reference/data-types/int-uint.md#uint-ranges))) — Logical value, it shows whether the interval is randomized. Interval always starts at the same time if it is not randomized. For example, an interval of 1 minute always starts at an integer number of minutes (i.e. it can start at 11:20:00, but it never starts at 11:20:01), an interval of one day always starts at midnight UTC. The randomized interval starts anytime. Values:
```